src/vmm/src/lib.rs

```diff
@@ -421,23 +411,63 @@ impl Vmm {
             .map(|handle| {
                 handle
                     .response_receiver()
-                    .recv_timeout(Duration::from_millis(400))
+                    .recv_timeout(Duration::from_millis(1000))
```

Comment:
Is this enough when the host is under CPU pressure ?

Reply:
Raised to `1s` to address exactly that. I am sure that in some extreme scenarios or on very slow hardware, `1s` might not be enough.

We can go even higher right now or tune this later if necessary.

`1000ms` is the value used to fix #1555